test/copy-filter.js

```diff
@@ -96,6 +96,12 @@ test('ignore: not junk when junk: false', util.testSinglePlatform(async (t, opts
   const targetDir = await copyDirToTempDirWithIgnores(t, opts)
   await assertFileNotIgnored(t, targetDir, 'subfolder/Thumbs.db')
 }))
+test('ignore: package manager lock files', util.testSinglePlatform(async (t, opts) => {
+  opts.dir = t.context.workDir
+  const targetDir = await copyDirToTempDirWithIgnores(t, opts)
+  await assertFileIgnored(t, targetDir, 'package-lock.json')
+  await assertFileIgnored(t, targetDir, 'yarn.lock')
```

Comment:
I don't think this test does what you want? As far as I know, `t.context.workDir` doesn't contain any package lock files before the copy happens. I suggest adding an empty `package-lock.json` and `yarn.lock` to a new fixture folder (`ignore-package-lock` perhaps, making sure they're added to git with `git add --force`) and then setting `opts.dir` to that path.

Reply:
Thanks for helping me out on my first open source issue! Yeah looking back this isn't testing what I thought it was. Changing the filename allows the test to pass regardless of filename.

I can see the code to add/set a fixture folder, but I can't see how you'd add files to that folder via `git add` in order to test that they aren't included.

Reply:
Sorry if that was unclear, I meant add empty (fake) package manager lock files to a new fixture folder via `git add --force` and then commit it to the branch, instead of trying to do that sort of thing via test code.

Reply:
Nope looking back that makes perfect sense - I misunderstood. I'll get those changes put in asap.

Reply:
Added a fixture folder with lockfiles, added to git and fixed up the test.